plugin/fallback/README.md

```diff
@@ -0,0 +1,73 @@
+# fallback
+
+## Name
+
+*fallback* - a plugin that sends queries to an alternate set of upstreams if the plugin
+chain returns specific error messages.
+
+## Description
+
+The *fallback* plugin allows an alternate set of upstreams be specified which will be used
+if the plugin chain returns specific error messages. The *fallback* plugin utilizes the *proxy*
+plugin (<https://github.com/coredns/coredns/tree/master/plugin/proxy>) to query the specified
+upstreams.
+
+As the name suggests, the purpose of the *fallback* is to allow a fallback when, for example, the
+desired upstreams became unavailable.
+
+## Syntax
+
+```
+{
+    fallback RCODE PROXY_PARAMS
+}
+```
+
+* `RCODE` is the string representation of the error response code. The set of valid error
+strings are defined as `RcodeToString` in <https://github.com/miekg/dns/blob/master/msg.go>
```

Comment:
I would list them here? Instead of pointing to a (large) file

Reply:
Yeah... but RcodeToString is a rather long list.

How about including a handful of typical example rcodes here, and refer to that link for the complete list?